frb_example/pure_dart/rust/build.rs

```diff
@@ -21,16 +21,21 @@ fn main() {
         wasm: true,
         dart_decl_output: Some("../dart/lib/bridge_definitions.dart".into()),
         dart_format_line_length: 120,
+        // (extra) c output path
+        c_output: Some(vec![
+            // each field should contain head file name
+            "./c_output_path/c_output.h".into(),
+        ]),
+        extra_c_output_path: Some(vec!["./c_output_path_extra/".into()]),
+
         // for other options use defaults
         ..Default::default()
     };
 
     // get opts from raw opts
-    let configs = config_parse(raw_opts);
+    let all_configs = config_parse(raw_opts);
 
-    // generation of rust api for ffi
-    let all_symbols = get_symbols_if_no_duplicates(&configs).unwrap();
-    for config in configs.iter() {
-        frb_codegen(config, &all_symbols).unwrap();
-    }
+    // generation of rust api for ffi (single block)
+    let all_symbols = get_symbols_if_no_duplicates(&all_configs).unwrap();
+    frb_codegen(&all_configs[0], &all_symbols).unwrap();
```

Comment:
would be great to assert there are only 1 elements in vec

Reply:
Refined in https://github.com/fzyzcjy/flutter_rust_bridge/pull/1024/commits/1217983475c2a047aa56593d4bb47da14ee354c1.